frontend/src/contexts/authContext.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, useState, useContext } from "react";
+
+interface AuthContextType {
```

Comment:
maybe move this to another file and import it

Reply:
I use AuthContextType just in this file